android-components/components/feature/downloads/src/main/java/mozilla/components/feature/downloads/share/ShareDownloadFeature.kt

```diff
@@ -147,7 +147,6 @@ class ShareDownloadFeature(
                 share(
                     contentType = internetResource.contentType,
                     filePath = download.canonicalPath,
-                    message = sanitizeLongUrls(internetResource.url),
```

Comment:
It looks like `sanitizeLongUrls` and `CHARACTERS_IN_SHARE_TEXT_LIMIT` won't be used in other places after this changes so they can be removed from code and tests.